README.md

```diff
@@ -0,0 +1,34 @@
+# Akeneo Labs Enhanced connector bundle
+
+An export bundle primarily designed to replace the [Dnd-MagentoConnectorBundle](https://github.com/Agence-DnD/DnD-MagentoConnectorBundle) and work with [PimGento](https://github.com/Agence-DnD/PIMGento).
```

Comment:
I would describe what the bundle does and after that speaking about PIMGento at the end of the description. Otherwise, you convey the sense that this bundle is only useful for PimGento, which is not the case. The product export can be interesting for a lot of projects.